session-replay/Tests/DatadogSessionReplayTests/Mocks/SRDataModelsMocks.swift

```diff
@@ -1089,6 +1089,15 @@ internal extension SRRecord {
         default: return nil
         }
     }
+
+    var viewportResizeData: SRIncrementalSnapshotRecord.Data.ViewportResizeData? {
+        if case let .incrementalSnapshotRecord(value: value) = self {
```

Comment:
This particular level of pyramid could be avoided if we move it to `extension SRIncrementalSnapshotRecord` below 😉 - not a blocker tho, we can see how it extends in the future.

```swift
extension SRIncrementalSnapshotRecord {
   var pointerInteractionData: SRIncrementalSnapshotRecord.Data.PointerInteractionData? { 
      // ...
   }

   var viewportResizeData: SRIncrementalSnapshotRecord.Data.ViewportResizeData? {
      // ...
   }
}
```